src/main/java/com/session/service/client/Http.java

```diff
@@ -56,10 +61,11 @@ private HttpPost createHttpPost(String host, String uri, String json, String ser
         return httpPost;
     }
 
-    public HttpDelete createHttpDelete(String host, String uri) {
+    private HttpDelete createHttpDelete(String host, String uri, String serviceAuthToken) {
         HttpDelete httpDelete = new HttpDelete(host + uri);
         httpDelete.setHeader(ACCEPT_HEADER_NAME, APPLICATION_JSON);
         httpDelete.setHeader(CONTENT_TYPE_HEADER, APPLICATION_JSON);
```

Comment:
Not really your code, but these DELETE requests do not send any body so there is no content in order to have a content type (unlike the POST which sends content). Therefore a Content-Type header shouldn't need to be set (unless the framework somehow moans without it which would be a shame). The Accept one above is needed though as content could be returned.

Reply:
Makes perfect sense, didn't notice this at first! Thanks.